anki/template/template.py

```diff
@@ -5,6 +5,7 @@
 from anki.template import hint; hint.install()
 
 clozeReg = r"(?s)\{\{c%s::(.*?)(::(.*?))?\}\}"
```

Comment:
It is not that the string is empty.  It is that the string has no sound in
it.
Additionally the string could be empty.


On Mon, Jan 25, 2016 at 4:45 PM, Julien Baley <notifications@github.com>
wrote:

> In anki/template/template.py
> <https://github.com/dae/anki/pull/116#discussion_r50761162>:
>
> > @@ -5,6 +5,7 @@
> >  from anki.template import hint; hint.install()
> >
> >  clozeReg = r"(?s)\{\{c%s::(.*?)(::(.*?))?\}\}"
>
> This is not part of the changes, but @dae <https://github.com/dae> it
> would help legibility if those groups were named (esp. since they're used
> quite far from their definition)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dae/anki/pull/116/files#r50761162>.
>